app/components/contribution-list/template.hbs

```diff
@@ -1,5 +1,5 @@
 {{#each contributions as |contribution|}}
-  <li data-contribution-id={{contribution.id}} class={{if contribution.confirmed "confirmed" "unconfirmed"}}>
+  <li data-contribution-id={{contribution.id}} class="{{contribution-status contribution}} {{if contribution.vetoed "vetoed"}}">
```

Comment:
This is a bit of a hack. It shouldn't required the `if` for the second class. But apparently with the helper, it doesn't bind the contribution property updates, so the value stays the same after setting the contribution `vetoed` to true. :/

Reply:
Mmh, indeed it looks like when passing objects to a helper, it doesn't recalculate when one of the object's properties changes.

I see two options:

1. Pass the two properties that the helper is using directly as parameters (`contribution-status contribution.vetoed contribution.confirmedAt`). 

2. Create a component for the contribution (e.g. `contribution-item`). This way the component can have a computed property for the class.

Reply:
Thanks!

I think version 2 is better, but that it should be done when refactoring the UI soon. I think we'll want to use something like `ember-boostrap` instead of our handrolled everything, because otherwise we spend too much time re-inventing the wheel for all the various UI components.

Reply:
I like version 2 better as well.